etl/src/main/java/org/pih/hivmigration/etl/sql/LabResultMigrator.groovy

```diff
@@ -0,0 +1,193 @@
+package org.pih.hivmigration.etl.sql
+
+class LabResultMigrator extends SqlMigrator {
+
+    @Override
+    def void migrate() {
+        executeMysql("Create staging table", '''
+            create table hivmigration_lab_results (
+              obs_id int PRIMARY KEY AUTO_INCREMENT,
+              source_patient_id int,
+              source_encounter_id int,
+              source_result_id int,
+              sample_id VARCHAR(20),
+              test_type VARCHAR(16), -- viral_load, CD4, tr, ppd, hematocrit
+              obs_datetime date,
+              value_numeric DOUBLE,
+              value_text VARCHAR(100),
+              value_boolean BOOLEAN,
+              vl_beyond_detectable_limit BOOLEAN,
+              vl_detectable_lower_limit DOUBLE,
+              tmp_vl_beyond_detectable_limit_concept int,
+              tmp_rapid_test_value int,
+              uuid char(38),
```

Comment:
Same comment here with uuid as my comment above with obs_id.  What exactly does this uuid represent?  We should be explicit, and possible have multiple obs fields (some of which might be null).  Eg. sample_id_uuid, quantitative_result uuid, qualitative_result_uuid, etc.

Reply:
Oh whoops, UUIDs are now added via the procedure, so this column isn't necessary at all any more.